test/e2e/private_link_test.go

```diff
@@ -34,7 +32,6 @@ var _ = Describe("UserLogin", Label("privatelink"), func() {
 	var testData *model.TestDataProvider
 
 	_ = BeforeEach(func() {
-		Eventually(kubecli.GetVersionOutput()).Should(Say(K8sVersion))
```

Comment:
I'm surprised that removing this doesn't cause any problems :)
I thought this is like "tell when you're ready flag".
Great if we don't need it at all 👌

Reply:
Yeah. Let's add it to the workflow file

Reply:
It's just a compression between the existing and expected versions of kubectl